src/content/post/2025-06-07-six-month-later.mdx

```diff
@@ -0,0 +1,36 @@
+---
+publishDate: 2025-06-07T12:00:00Z
+author: ildyria
+title: 'Six months later on version 6'
+excerpt: We look back on what we have accomplished over those past six months and what is next on the roadmap?
+image: https://images.unsplash.com/photo-1522635922534-827978bf2a2f?q=80&w=3540&auto=format&fit=crop&ixlib=rb-4.1.0&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D
+category: Release Notes
+tags:
+  - lychee
+  - blabla
+  - Supporter Edition
+---
+
+First of all, I apologize for the silence over the past few months.
+Nevertheless, just because I was not posting does not mean I wasn’t working on Lychee, as shown by the multiple releases since then: 6.3, 6.4, 6.5, and 6.6, including several intermediate patches.
+
+Six months ago, I wrote a [post](https://lycheeorg.dev/2024-12-13-what-is-next/) about the future of Lychee and my vision for it.
+Looking at the todo list I wrote back then, one might think that we have not accomplished much.
+The main reason is that, rather than rushing to add many features that might break often,
+Lychee focuses on stability and quality. We want to ensure that the foundations are solid before adding new features.
+This is why we have been focusing on the `Must have` items:
+
+- User groups have been implemented. Integration with access control is still pending, but coming soon.
+- Improved integration with OAuth and SSO.
+- Permission propagation has been implemented.
+- Mass sharing v4 has been implemented, allowing users to share multiple albums and photos on a single page.
+- Settings have been completely refactored with a new layout, allowing grouping and hiding complexity.
+- Localization has been completely reset and improved. We now support Weblate, allowing anyone to contribute translations.
```

Comment:
Weblate could possibly be a link, but I have no strong preference - just a thought.